websockets/test_client_server.py

```diff
@@ -401,6 +402,16 @@ def test_server_shuts_down_during_connection_handling(self):
         # Websocket connection terminates with 1001 Going Away.
         self.assertEqual(self.client.close_code, 1001)
 
+    def test_invalid_status_error_during_client_connect(self):
+        self.start_server(klass=ForbiddenWebSocketServerProtocol)
+        with self.assertRaises(InvalidStatus) as context:
```

Comment:
Sure. Note that `context` is also used once in test_protocol.py. Incidentally, the [Python docs](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) use `cm` in their example code snippet.